pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+{% include nav.html %} {% include css.html %}
+
 # Analysis
+---
+	
+The visualization on this page represents the documented interference
+on the slave trade by privateers and pirates between 1699-1827. The primary 
+geographical focus is in the Caribbean and West Indies region. These regions 
+are where the most slave trade activity took place. The original hypothesis 
+for this research is that privates and privateers had a significant effect on 
+the slave trade. Pirates and privateers caused fiscal strains on the slave t
+rade market and economically effected countries that participated in the slave 
+trade. As per the documented information, the slave trade was extremely expansive
+that the incursion by privateers had little to no effect on the market itself. 
+
+The images do raise questions to the effects privateers and pirates had on smaller
+countries. These smaller states would not have had such availability to the slave 
+trade if not provided by the captive ships. The images below demonstrate the 
+disparities and theory.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/1.%20Origin%20Ports.png)
+	
+This image represents most of the documented ports where slave 
+ships originated. To the left (black dots) the image is of all the ships
+that departed and were never held captive and taken over by privateers or 
+pirates. The image to the right (purple dots) shows the ports where ships
+launched and eventually were taken over by privateers or pirates. The
+purpose of these images is to show that at this point there is no significance
+between pirated ships and ships that completed their voyage without being 
+impeded. The theory is that to this point pirates and privateers were more 
+opportunistic scavengers of ships and less methodical in their selection
+
+---
+
+![image]({{ site.baseurl}}/assets/images/2.%20To%20slave%20ports.png)
+
+The path of travel represents paths as the crow flies, although 
+traveling ships make the paths. The quantity of activity is a representation 
+of over one hundred years but it is easy to see the differences between the 
```

Comment:
I wonder if it might be interesting (if you had had more time), to see this visualization in several variations... filtering the 100+ years to several chunks, say of 40ish years (or some other logical subdivision). That would allow a significant comparison.